frontend/src/pages/Home/components/Criteria/Criteria.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import slugify from 'slugify';
+import classNames from 'classnames';
+import { useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { updateCategory, updateUrl, updateDesigner, updateSort } from './filterSlice';
+import './Criteria.scss';
+
+interface IProps {
+  title: string;
+  criteria: string[];
+  type: string;
+}
+const Criteria: React.FC<IProps> = (props: IProps) => {
```

Comment:
ex: const Criteria: React.FC<CriteriaProps> = (props: CriteriaProps): ReactElement => {
or
const Criteria: React.FC<CriteriaProps> = ({ type, title, criteria }): ReactElement => {
so you don't have to destruct it below